packages/proto-signing/src/directsecp256k1hdwallet.ts

```diff
@@ -91,7 +91,11 @@ export class DirectSecp256k1HdWallet implements OfflineDirectSigner {
   private readonly accounts: readonly Secp256k1Derivation[];
 
   protected constructor(mnemonic: EnglishMnemonic, options: DirectSecp256k1HdWalletConstructorOptions) {
-    const { seed, hdPaths, prefix } = { ...defaultOptions, ...options };
+    const { seed, hdPaths, prefix } = {
```

Comment:
Could you turn those into 3 simple `const`s? No need for the surrounding object anymore.

Reply:
8bed00ed1

test on its way